README.md

```diff
@@ -112,19 +112,19 @@ Fluent Bit Operator supports `docker` as well as `containerd` and `CRI-O`. `cont
 If your container runtime is `docker`
 
 ```shell
-helm install fluentbit-operator  --create-namespace -n kubesphere-logging-system charts/fluent-operator/ --set containerRuntime=docker
+helm install fluent-operator  --create-namespace -n kubesphere-logging-system charts/fluent-operator/ --set containerRuntime=docker
```

Comment:
The namespace should use an env variable to set before any helm install cmd, and reference this env in the following helm install cmd

Reply:
helm does not initially support the environment variable approach for security reasons. Please see：https://github.com/helm/helm/issues/944 . helm3 can use `set` to set environment variables. But I don't think there is much difference between this way and the previous one. Perhaps we could create a `namespace.yaml`  file and change its namespace by setting the value in `value.yaml`?

Reply:
We can use downward API with `{{ .Release.Namespace }}` to tell the CRs what's namespace the helm used.

Reply:
Ok, maybe we needn't change the namespace as a helm parameter.
My concern is this is a community project now, the kubesphere namespace is not appropriate anymore.

We might need to use a namespace like `fluent` for all helm, YAML and samples

Reply:
OK，I will revise and test as soon as possible